vibe_code_apps_22/eplus_gym/rl/live_day_worker.py

```
@@ -102,6 +118,15 @@ def factory():
         "simulator": "LIVE_ENERGYPLUS",
     }
     (ep_dir / "reward.json").write_text(json.dumps(payload, indent=2) + "\n", encoding="utf-8")
+    err = ep_dir / "eplus" / "eplusout.err"
+    if not err.is_file():
+        found = list(ep_dir.rglob("eplusout.err"))
+        err = found[0] if found else err
+    gate = parse_eplus_err(err)
+    payload["eplus_quality"] = gate
+    payload["lookback_days"] = lb
+    payload["n_all_rows"] = int(len(result.get("all_rows") or []))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)